addons/hr_recruitment_survey/wizard/survey_invite.py

```diff
@@ -1,9 +1,8 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models, _
+from odoo import _, fields, models
 from odoo.tools.misc import clean_context
 
```

Comment:
Also why?

Reply:
coding guidelines:
https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#imports

Reply:
And, what does it say? :innocent:

Reply:
I agree with @dasz-odoo. `_` is generally placed before letters in alphabetical ordering (and so the inconsistency in the docs is probably a mistake).
#SuchAReview@SMLearn

Reply:
I remember reading somewhere that the underscore should be in front of everything, but I can't find the source anymore. The majority of the code imports _ at the end of the string, so I'll revert that specific change and stick to the docs.

Reply:
Well, if it's alphabetical, then `_` comes before letters indeed.  But that's being said, our code base is not consistent. Good half of the time it's before, good half of the time it's after the letters. So this change would have been change just because of change. 

For the other change, there should be two lines before the class declaration. So why are you removing it? Moreover, stylistic changes when the codebase is not consistent are not that beneficial.

For the third comment, (tde) mentioned it, I am wondering as well,  why are you having `auto_commit=True`? As you have not changed it, I am guessing you have a reason, so feel free to let me know as well.

A reference about modifying existing file - https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#:~:text=When%20modifying%20existing%20files%20in%20master%20(development)%20version%20apply%20those%20guidelines%20to%20existing%20code%20only%20for%20modified%20code%20or%20if%20most%20of%20the%20file%20is%20under%20revision